app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -0,0 +1,105 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.ProductNotFountException;
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.domain.ProductRepository;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+
+/**
+ * {@code ProductService} 클래스는 {@code Product}에 대한 비즈니스 로직을 처리합니다.
+ *
+ * @see Product
+ */
+@Service
+@Transactional
+public class ProductService {
+    final ProductRepository productRepository;
+
+    public ProductService(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    /**
+     * 새로운 product를 생성합니다.
```

Comment:
리턴값에 대해 설명하는 것이 중요합니다.
```suggestion
     * 새로운 product를 생성해 리턴합니다.
```

Reply:
리턴값이 있는 경우 리턴값을 설명하는게 핵심이겠군요 :)